setup.py

```diff
@@ -351,8 +377,19 @@ def check_file(f):
     ATEN_LIB = os.path.join(lib_path, 'libATen.1.dylib')
     NCCL_LIB = os.path.join(lib_path, 'libnccl.1.dylib')
 
-# static library only
-NANOPB_STATIC_LIB = os.path.join(lib_path, 'libprotobuf-nanopb.a')
+if IS_WINDOWS:
+    TH_LIB = os.path.join(lib_path, 'TH.lib')
+    THS_LIB = os.path.join(lib_path, 'THS.lib')
+    THC_LIB = os.path.join(lib_path, 'THC.lib')
+    THCS_LIB = os.path.join(lib_path, 'THCS.lib')
+    THNN_LIB = os.path.join(lib_path, 'THNN.lib')
+    THCUNN_LIB = os.path.join(lib_path, 'THCUNN.lib')
+    THPP_LIB = os.path.join(lib_path, 'THPP.lib')
+    ATEN_LIB = os.path.join(lib_path, 'ATen.lib')
+    _C_LIB = 'build/temp.win-amd64-' + str(sys.version_info[0]) + '.' + str(
+        sys.version_info[1]) + '/Release/torch/csrc/_C.cp' + str(
+            sys.version_info[0]) + str(sys.version_info[1]) + '-win_amd64.lib'
```

Comment:
This seems super fragile. I'm pretty sure there is a way to get this info from `distutils`